source/feature-renames.test.ts

```diff
@@ -0,0 +1,31 @@
+
+import {execSync} from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+import {test, expect} from 'vitest';
+
+import featureRenames from './feature-renames.json';
+
+const oldNames = Object.keys(featureRenames);
+const newNames = Object.values(featureRenames);
+
+test('Old feature names cannot appear anywhere in the repo', () => {
+	for (const oldName of oldNames) {
+		const grep = `rg -l "[^-]${oldName}[^-]" -g !feature-renames.json -g !package-lock.json`;
```

Comment:
It requires ripgrep to be installed. 🤷‍♂️ 

Not required for building the extension, so not a problem. But actually the test passes anyway when it's missing 😅